src/test/selectors/generateChannelLinkTest.js

```diff
@@ -0,0 +1,16 @@
+import { generateChannelLink } from '../../selectors/generateChannelLink.js';
+
+describe('generateChannelLink', function() {
+  it('should return appended wordpress uri', function() {
+    expect(
+      generateChannelLink('htttp://www.example.com?Param1=param', 'wordpress')
+    ).toBe(
+      'htttp://www.example.com?Param1=param&channel=Integration:%20WordPress'
+    );
+  });
+  it('should return new cpanel uri', function() {
+    expect(generateChannelLink('htttp://www.example.com', 'cpanel')).toBe(
+      'htttp://www.example.com?channel=Integration:%20CPanel'
+    );
+  });
```

Comment:
Actually the tests make sense because it tests for params and also we're returning specific things based on the integration. I think removing the magento and someNewChannel test makes sense but the rest should stay.